modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -790,16 +838,6 @@
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform.dmi'
 	icon_state = "plasmaman_jax"
 
-// Donation reward for Raxraus
-/obj/item/clothing/under/rax_turtleneck
-	icon = 'icons/obj/clothing/under/security.dmi'
-	worn_icon = 'icons/mob/clothing/under/security.dmi'
-	name = "black turtleneck"
-	desc = "A stylish black turtleneck."
-	icon_state = "hosalt"
-	inhand_icon_state = "bl_suit"
-	alt_covers_chest = TRUE
```

Comment:
How's this related to the rest?

Reply:
It's a removal of other donator items that Raxraus weren't using, which make place for the Caligra donator items.